### PR TITLE
HARP-7356: Added TextElementsRendererTest

### PR DESCRIPTION
**Pull request by ninok**

### CODE REVIEW

File: @here/harp-mapview/lib/text/TextElementsRenderer.ts [hunk 1]
```diff
@@ -666,6 +666,227 @@ export class TextElementsRenderer {
         });
     }
 
+    renderTextElements(
```
Comment (ninok) [inline @here/harp-mapview/lib/text/TextElementsRenderer.ts]: This method was just moved up b/c it is now public and we have a tslint rule that public methods have to come first.

File: @here/harp-mapview/test/TextElementsRendererTest.ts [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        assert(renderedTextElements[0].textRenderState!.isFadingIn());
+
+        const lastOpacity = renderedTextElements[0].textRenderState!.opacity;
+
+        // Third frame. Label should be semi-transparent and still fading in.
+        renderedTextElements.splice(0);
+        numRenderedTextElements = textElementsRenderer.renderTextElements(
+            textElements,
+            time + 800,
+            frameNumber + 8,
```
Comment (nzjony) [inline @here/harp-mapview/test/TextElementsRendererTest.ts]: should this be +3?

File: @here/harp-mapview/test/TextElementsRendererTest.ts [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+            secondChanceTextElements
+        );
+        expect(numRenderedTextElements).to.be.equal(1);
+        expect(renderedTextElements.length).to.be.equal(1);
+        expect(renderedTextElements[0].textRenderState!.opacity)
+            .to.be.greaterThan(lastOpacity)
+            .to.be.lessThan(1);
+        assert(renderedTextElements[0].textRenderState!.isFadingIn());
+
+        // Forth frame. Label should be visible and fading should be finished.
```
Comment (nzjony) [inline @here/harp-mapview/test/TextElementsRendererTest.ts]: Why does it take 4 frames to fade in? I tried to find where this is configured but couldn't, is it some default?

Comment (ninok) [reply @here/harp-mapview/test/TextElementsRendererTest.ts]: It is not the number of frames that matter, but the time. It is configurable but by default it is:
`export const DEFAULT_FADE_TIME = 800;` defined in TextElement.ts`
I will use this constant to make things more clear.

File: @here/harp-mapview/test/TextElementsRendererTest.ts (hunk 2 above)
Comment (ninok) [reply @here/harp-mapview/test/TextElementsRendererTest.ts]: True. I actually don't know if the frameNumber is used for anything. I think the only requirement is, that it is monotonically increasing

**Comment by ninok** [comment]
@heremaps-bot retest this please

**Pull request merged**